Cauldron/Assets/Scripts/DI/GameplayInstaller.cs

```diff
@@ -72,7 +72,7 @@ private void BindGameplay()
 
         private void Initialize()
         {
-            gameDataHandler.Init(mainSettings, encounterDeck, nightEvents, dataPersistenceManager, soDictionary);
+            gameDataHandler.Init(mainSettings, encounterDeck, dataPersistenceManager, soDictionary);
```

Comment:
That fixes the bug with some story events getting in the new game from the old one? Right? Right?